onnxruntime/core/graph/contrib_ops/collective_defs.cc

```diff
@@ -106,22 +114,32 @@ void RegisterCollectiveOps() {
              "T")
       .Input(2,
              "fc1_experts_weights",
-             "3D input tensor with shape (local_num_experts, hidden_size, inter_size)",
+             "3D input tensor with shape (local_num_experts, hidden_size, local_inter_size)",
              "T")
       .Input(3,
```

Comment:
The change is not backward compatible. Is there customer using it in 1.17?

Reply:
no customer is using it so it should be safe